frontend/src/components/OdhDocCard.tsx

```diff
@@ -67,6 +76,7 @@ const OdhDocCard: React.FC<OdhDocCardProps> = ({ odhDoc, favorite, updateFavorit
     e.preventDefault();
     launchQuickStart(odhDoc.metadata.name, qsContext);
     makeCardVisible(odhDoc.metadata.name);
+    fireResourceAccessedEvent(odhDoc.metadata.name, odhDoc.metadata.type)();
```

Comment:
Do we care if it is a `start` vs `continue` vs `restart`?

Reply:
I don't see this in the JIRA issue. Hi, @anishasthana do we care about whether the user starts the quick start, or continues, or restarts? Do we also need to send that information?